Prefetching-Library/README.md

```diff
@@ -123,3 +123,41 @@ After implementing a strategy, please consider sharing it by creating a PR to th
 To create a new build with the modifications implemented in the library, click on `Build > Make Project`.
 Android Studio places new builds in the directory [android_prefetching_lib/build/outputs/aar/](android_prefetching_lib/build/outputs/aar).
 To use this build, repeat the `Import Library` instructions in the `Installation` section.
+
+### Debug the database data
+
+#### Using DB Browser for SQLite
```

Comment:
Turned out to be a great way to test a few queries without reimporting the library